docker-compose.yml

```diff
@@ -11,6 +11,11 @@ services:
       - 5432:5432
     expose:
       - 5432
+    volumes:
```

Comment:
what's the volumes stuff for?

Reply:
This allows timescale to persist beyond the lifetime of the router.  Before we just avoided every properly shutting down service we just pulled the plug.  With this timescale already exists when running docker compose up.  It will save about 2 min boot time (down to about 10 seconds) and will prevent anyone from accidentally losing all data.  I'll add a note in the Readme about this.

Reply:
🔥